### PR TITLE
reduce dependence on StatsBase (now only needed for binomial distrib…

### PR DESCRIPTION
**Pull request by CarloLucibello**
…ution)

### CODE REVIEW

File: src/utils.jl [hunk 1]
```diff
@@ -1,9 +1,8 @@
 """
+sample!([rng,] a, k; exclude = ())
+
```
Comment (sbromberger) [inline src/utils.jl]: Why are we allowing what appears to be a vector of random number generators here?

Comment (jpfairbanks) [inline src/utils.jl]: I think that is denoting rng is an optional argument. There is a second method of this function that does not take rng as an argument and uses the standard one.

Comment (CarloLucibello) [inline src/utils.jl]: exactly

Comment (sbromberger) [inline src/utils.jl]: Man, this is what I get for looking at this before caffeine. This is in the docstring, not the code. I get it. :)

**Pull request merged**